api/users.go

```diff
@@ -75,6 +86,10 @@ func PostUser(c *gin.Context) {
 		return
 	}
 
+	if in.Login == "" {
+		c.AbortWithError(422, errors.New("User's login can't be empty"))
```

Comment:
Generally I'm mostly using `fmt.Errorf` instead of `errors.New`, but that is more like a personal taste.